### PR TITLE
Clarify info about tiered compilation in "what's new" doc

### LINKED ISSUE #15143: Link between 'Quick JIT (tier 0 jitted code)' and Tiered Compilation unclear
It says that in dot net core 3.0 tiered compilation is enabled by default and then explains the benefits of it. Then, out of the blue, it says how to enable 'Quick JIT (tier 0 jitted code)' without explaining if that is the same thing as tiered compilation or some extra improvement, nor any guidance on when we would want to enable it

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d458030b-3efb-197f-6ac7-9fd4e2296218
* Version Independent ID: 6d01409e-fca6-10a3-2def-13ce28c9b210
* Content: [What's new in .NET Core 3.0](https://docs.microsoft.com/en-us/dotnet/core/whats-new/dotnet-core-3-0#feedback)
* Content Source: [docs/core/whats-new/dotnet-core-3-0.md](https://github.com/dotnet/docs/blob/master/docs/core/whats-new/dotnet-core-3-0.md)
* Product: **dotnet-core**
* GitHub Login: @Thraka
* Microsoft Alias: **adegeo**

### PR DESCRIPTION
**Pull request by kouvel**
## Summary

Clarify what Quick JIT means, that it is enabled in .NET Core 3.0 by default, and describe how to disable it if there are issues.

Fixes https://github.com/dotnet/docs/issues/15143

### CODE REVIEW

**Comment by kouvel** [comment]
CC @noahfalk

**Comment by kouvel** [comment]
CC @mairaw

File: docs/core/whats-new/dotnet-core-3-0.md [hunk 1]
```diff
@@ -112,20 +112,33 @@ For more information about the IL Linker tool, see the [documentation](https://a
 
 The main benefit of TC is to enable (re-)jitting methods with a lower-quality-but-faster tier or a higher-quality-but-slower tier. This helps increase performance of an application as it goes through various stages of execution, from startup through steady-state. This contrasts with the non-TC approach, where every method is compiled a single way (the same as the high-quality tier), which is biased to steady-state over startup performance.
 
-To enable Quick JIT (tier 0 jitted code), use this setting in your project file:
+When TC is enabled, during startup for a method that is called:
+
+- If the method has AOT-compiled code (ReadyToRun), the pregenerated code will be used
```
Comment (Thraka) [inline docs/core/whats-new/dotnet-core-3-0.md]:
```suggestion
- If the method has AOT-compiled code (ReadyToRun), the pregenerated code will be used.
```

**Review by Thraka — CHANGES_REQUESTED** [review]
Missing period. Otherwise it's ready to go. Thanks!

**Review by Thraka — APPROVED** [review]
Looks good, thanks!

**Pull request merged**